src/Repositories/RedisRepository.php

```diff
@@ -77,32 +77,60 @@ public function partitionsWithCount($queue)
         return $partitions;
     }
 
-    public function jobs($partition)
+    public function totalJobsCount($queues)
     {
         $prefix = $this->getPrefix();
         $redis = $this->getConnection();
-        $per_page = request('limit', 100);
 
-        $jobs = $redis->lrange($partition, 0, $per_page + 1);
+        $jobs_count = 0;
+
+        foreach($queues as $queue) {
+            foreach($this->partitions($queue) as $partition) {
+                $jobs_count += $redis->llen("{$prefix}:{$queue}:{$partition}");
+            }
+        }
+        return $jobs_count;
+    }
+
+    public function jobs($partition)
```

Comment:
are you expecting a redis key here? it shouldn't be like this. redis key should be always generated.